src/bci_build/package/postgres/README.md.j2

```diff
@@ -0,0 +1,126 @@
+
+#### POSTGRES_HOST_AUTH_METHOD
+
+This optional variable can be used to control the `auth-method` for host connections for all databases, users, and addresses. If unspecified, the `scram-sha-256` password authentication is used.
+
+On an uninitialized database, this populates `pg_hba.conf` via this approximate line:
+
+`echo "host all all all $POSTGRES_HOST_AUTH_METHOD" >> pg_hba.conf`
+
+For more information about possible values and their meanings, refer to the PostgreSQL documentation on [password Authentication](https://www.postgresql.org/docs/14/auth-password.html) and [pg_hba.conf](https://www.postgresql.org/docs/14/auth-pg-hba-conf.html).
+
+**Note 1:** If you set `POSTGRES_HOST_AUTH_METHOD` to `trust`, then `POSTGRES_PASSWORD` is not required since it allows anyone to connect without a password.
+
+**Note 2:** If you set `POSTGRES_HOST_AUTH_METHOD` to an alternative value, you might need additional `POSTGRES_INITDB_ARGS` for the database to initialize correctly.
+
+#### PGDATA
+
+The value for this variable is `/var/lib/pgsql/data`. This location is a volume and another location is currently not supported.
+
+#### Sensitive information
+
+As an alternative to passing sensitive information via environment variables, `_FILE` can be appended to `POSTGRES_INITDB_ARGS`, `POSTGRES_PASSWORD`, `POSTGRES_USER`, and `POSTGRES_DB` environment variables. This makes the initialization script load the values for those variables from files present in the container. To, e.g., pass the password securely, you can store the password in a secret `postgress-pw` and launch the container as follows:
+
+```ShellSession
+$ podman run -it --rm
+    -p 5432:5432 \
+    -e POSTGRES_PASSWORD_FILE=/run/secrets/postgress-pw \
+    --secret postgress-pw \
+    -v /path/to/data:/var/lib/pgsql/data:Z \
+    {{ image.pretty_reference }}
+```
+
+### Health, liveness, and readiness
+
+There is one explicit health check added to the container image. This check executes the `pg_isready` for host `localhost` and port `5432`.
+
+The utility [pg_isread](https://www.postgresql.org/docs/current/app-pg-isready.html) checks the connection status of the server, and the exit status specifies the result of the connection check.
+
+**Warning:** Scripts in `/docker-entrypoint-initdb.d` are only executed when you start the container with an empty data directory. Any pre-existing databases are left untouched when the container starts.
+
+**Note:** If one of the `/docker-entrypoint-initdb.d` scripts fails, and the container restarts with the already initialized data directory, it aborts the running scripts.
```

Comment:
Wrong section? Looks like it belongs into `Initialization scripts`

Reply:
Agree, it makes more sense to move.